lib/ansiblelint/__init__.py

```diff
@@ -23,11 +23,12 @@
 import os
 import re
 import sys
+import codecs
 
 import six
 
 import ansiblelint.utils
-import codecs
+from ansiblelint import skip_utils
```

Comment:
Can we use relative imports?
(this may require `from __future__ import absolute_imports`)